src/caffe/proto/caffe.proto

```diff
@@ -154,7 +154,7 @@ message SolverParameter {
   optional string lr_policy = 8; // The learning rate decay policy.
   optional float gamma = 9; // The parameter to compute the learning rate.
   optional float power = 10; // The parameter to compute the learning rate.
-  optional float momentum = 11; // The momentum value.
+  optional float momentum = 11 [default = 0.9]; // The momentum value.
```

Comment:
https://github.com/BVLC/caffe/blob/master/src/caffe/test/test_gradient_based_solver.cpp#L334
Default momentum is 0.0.